apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1588,22 +1588,50 @@
                     "type" : "object",
                     "additionalProperties" : false,
                     "properties" : {
-                        "enabled": {
+                        "enabled" : {
                             "description": "If true, node will perform garbage collection of account state, removing unreachable merkle patricia nodes and free some disk space",
                             "type": "boolean",
                             "default": false
                         },
+                        "from_start" : {
```

Comment:
I don't think this property name is technically correct or I'm expecting something a bit different from this functionality. Should be something more like "during_sync", "desync" or something like that?

Also, the description should mention how "in sync" is defined for this feature to work, percentage or somehow else ?